### DISCUSSION TITLE
Is it possible to schedule publishing events to appear on the page?

### DISCUSSION THREAD

**Discussion by tiicaa** [liedekef/events-made-easy#150] [Q&A]
Hello, I would like to be able to schedule events, in certain periods there are already all the scheduling dates of the month for example, but the user needs to see this gradually. Is this possible somehow in the plugin? I haven't found anything unless inserting the shortcode inside a page - program it and call it in the section where this event should appear, that seems a bit taxing is there any other way?

**Comment by liedekef** [comment]
See the scope param for [eme_events]: https://www.e-dynamics.be/wordpress/category/documentation/6-shortcodes/eme_events/ 
However, there's no event-option to publish an event on a certain date.
